### DISCUSSION TITLE
完善的建议

### DISCUSSION THREAD

**Discussion by adoin** [ikun-svelte/ikun-ui#197] [Ideas]
1、做个抽屉组件，铁山靠把抽屉顶出来，然后消失。
2、loading就把组件库的LOGO那个篮球弄个上下的动画先。
3、IKUN配色：IKun灰 IKun黑 IKun白 和篮球黄。应更广泛使用。

**Comment by codingframer** [comment]
> 1、做个抽屉组件，铁山靠把抽屉顶出来，然后消失。 2、loading就把组件库的LOGO那个篮球弄个上下的动画先。 3、IKUN配色：IKun灰 IKun黑 IKun白 和篮球黄。应更广泛使用。

附议

**Comment by cevin** [comment]
- select ikun在右上角拍球 然后跟着篮球下坠展开菜单
- radio 应该明暗篮球
- loading 应该是居中拍球
- input 获取焦点应该铁山靠篮球从左到右，失去焦点应该再回去

**Comment by LegendarySSS** [comment]
> 1、做个抽屉组件，铁山靠把抽屉顶出来，然后消失。 2、loading就把组件库的LOGO那个篮球弄个上下的动画先。 3、IKUN配色：IKun灰 IKun黑 IKun白 和篮球黄。应更广泛使用。

你们兼职石油兵😭

**Comment by LikeDreamwalker** [comment]
灰色底色+黑白橙高亮
感觉是material design版的pornhub风格

**Comment by progerchai** [comment]
> * select ikun在右上角拍球 然后跟着篮球下坠展开菜单
> * radio 应该明暗篮球
> * loading 应该是居中拍球
> * input 获取焦点应该铁山靠篮球从左到右，失去焦点应该再回去

意见中肯！

**Comment by linyangxin0** [comment]
一人血书求支持

**Comment by ikun-ProMax** [comment]
ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @ikun-ui/docs@0.0.2 docs:dev: `vitepress dev`
Exit status 1   一直报这个错什么鬼额

**Comment by baiwusanyu-c** [comment]
> ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @ikun-ui/docs@0.0.2 docs:dev: `vitepress dev` Exit status 1 一直报这个错什么鬼额

给出完整报错哦，尝试先 `pnpm run i`，`pnpm run build` 然后再 `pnpm run dev:docs`, 目前开发还是得先构建一遍...我也没找到合适的解决方法

**Comment by ikun-ProMax** [comment]
> > ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @ikun-ui/docs@0.0.2 docs:dev: `vitepress dev` Exit status 1 一直报这个错什么鬼额
> 
> 给出完整报错哦，尝试先 `pnpm run i`，`pnpm run build` 然后再 `pnpm run dev:docs`, 目前开发还是得先构建一遍...我也没找到合适的解决方法
? Please choose a version of "@ikun-ui/utils" from this list: 0.0.8
error An unexpected error occurred: "https://registry.npm.taobao.org/@ikun-ui%2fplay: Not found".
info If you think this is a bug, please open a bug report with the information provided in "C:\\Users\\Administrator\\Desktop\\ikun-ui\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command. 安装不到这个play模块好像

**Comment by baiwusanyu-c** [comment]
这是一个monorepo，使用 pnpm 安装

**Comment by ikun-ProMax** [comment]
可以了

**Comment by ikun-ProMax** [comment]
推荐多整点kunkun元素 个性化ui

**Comment by Kun8018** [comment]
下蛋🥚？

**Comment by toplyb** [comment]
> 1、做个抽屉组件，铁山靠把抽屉顶出来，然后消失。 2、loading就把组件库的LOGO那个篮球弄个上下的动画先。 3、IKUN配色：IKun灰 IKun黑 IKun白 和篮球黄。应更广泛使用。

Switch 组件也可以用，开关的那个组件换成个🏀，然后再加上铁山靠的动画 😂
